src/scripts/modules/gradebook.js

```diff
@@ -12,10 +12,12 @@
     const TABLE_TEMPLATE = `
       <h6 id='path-title'></h6>
 
-      <table class="table table-condensed">
-        <thead></thead>
-        <tbody></tbody>
-      </table>
+      <div class="grades">
```

Comment:
We'd prefer to use more semantic tags. I know it may seem very similar, but I'd prefer to see a `<section>` here if you need to contain the table.